composer.json

```diff
@@ -6,30 +6,37 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
+        "php": ">=7.4",
         "ext-json": "*",
-        "drupal/core": "^8.9 || ^9"
+        "drupal/core": "^9.2",
+        "drupal/ctools": "^3.7",
+        "drupal/file_link": "2.0.4",
+        "drupal/pathauto": "^1.8",
+        "drupal/token": "^1.10"
     },
     "require-dev": {
+        "behat/behat": "^3.10",
         "composer/installers": "~1.5",
-        "drupal/core-composer-scaffold": "^8.9 || ^9",
+        "drupal/core-composer-scaffold": "^9.2",
+        "drupal/core-dev": "^9.2",
         "drupal/config_devel": "~1.2",
         "drupal/drupal-extension": "^4.0",
         "drupal/geocoder": "^3.0",
         "drupal/geofield": "^1.0",
         "drupal/json_field": "~1.0-rc4",
         "drush/drush": "~10.3",
-        "nikic/php-parser": "^3.1.5",
-        "drupaltest/behat-traits": "dev-GH-6",
+        "egulias/email-validator": "^2.1.22 || ^3.0",
+        "nikic/php-parser": "^4.12.0",
```

Comment:
Do we still need this dependency?

Reply:
We do need as long as we support PHP7.4.